core/serv/src/main.rs

```diff
@@ -598,6 +641,27 @@ async fn me(id: Identity) -> impl Responder {
     web::Json(id)
 }
 
+#[actix_web::post("/_gsb/{service:.*}")]
+async fn forward_gsb(
+    id: Identity,
+    web::Path(service): web::Path<String>,
+    data: web::Json<serde_json::Value>,
+) -> impl Responder {
+    use ya_service_bus::untyped as bus;
+    log::debug!(target: "gsb-bridge", "called: {}", service);
+    let data = flexbuffers::to_vec(data.into_inner()).map_err(actix_web::error::ErrorBadRequest)?;
```

Comment:
It'd better to use `ya_service_bus::serialization::to_vec` and `ya_service_bus::serialization::from_slice`:
- `ya-service-bus` might be compiled with a different serialization method
- no need to keep library versions in sync

Reply:
fixed